### DISCUSSION TITLE
Exclude cronjob for external drives

### DISCUSSION THREAD

**Discussion by TommasoCostanzo** [nextcloud/all-in-one#2699] [Questions] [answered]
Hello nextcloud AIO team, I am having an hard time to avoid my disks to spin up approximately every hour. I have two USB disks that are mounted as local external drives. The files on these disks are not changing very frequently and I also do not access to them very frequently (maybe once a day). However, the discs keep waking up every hour when the cronjob run in the background. This is vary sub-optimal for the wear of spinning disks and also in terms of power consumption.

So far I have tried from the interface to set to "never" the "check for changes" and removed the previews. However, the disks keeps waking up. 

So, I would like to ask if there is a solution to this problem. I would be more then happy to completely exclude the external drive from the cronjob, and taking care of them manually via occ commands.

**Comment by szaimen** [comment, marked as answer]
Hi, please ask this question at https://help.nextcloud.com thanks!
